### PR TITLE
fix(labels): consider orientation when determining largest rect

### PR DESCRIPTION
**Pull request by quanho**

### CODE REVIEW

File: packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js [hunk 1]
```diff
@@ -253,6 +255,22 @@ describe('labeling - bars', () => {
       expect(p.bounds).to.equal(rects.biggest);
     });
 
+    it('should find largest rect as fallback, horizontal, and when all rects have the same height', () => {
+      rects.inside.height = rects.outside.height = rects.biggest.height = rects.meh.height = 100;
```
Comment (cbt1) [inline packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js]: Don't use this kind of syntax. Declare a const with desired value and assign it to each property on a separate row.

File: packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js [hunk 2]
```diff
@@ -267,6 +285,22 @@ describe('labeling - bars', () => {
       expect(p.bounds).to.equal(rects.biggest);
     });
 
+    it('should find largest rect as fallback, vertical, and when all rects have the same width', () => {
+      rects.inside.width = rects.outside.width = rects.biggest.width = rects.meh.width = 100;
```
Comment (cbt1) [inline packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js]: Don't use this kind of syntax. Declare a const with desired value and assign it to each property on a separate row.

File: packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js (hunk 1 above)
Comment (quanho) [reply packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js]: Fixed!

File: packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js (hunk 2 above)
Comment (quanho) [reply packages/picasso.js/src/core/chart-components/labels/strategies/__tests__/bars.spec.js]: Fixed!

**Pull request merged**